java/src/main/java/com/saucelabs/simplesauce/Platforms.java

```diff
@@ -1,7 +1,17 @@
-package com.saucelabs.simple_sauce;
+package com.saucelabs.simplesauce;
 
 public class Platforms
 {
+	public static final String WINDOWS_10 = "Windows 10";
```

Comment:
I'm starting to envision how we could move this to a kind of config file that would be generated based on a call to the Platform Configurator. I opened issue-17 about this. It's not a requirement for beta, I just wanted to throw that out there.